src/tests/functional/plugin/shared/include/behavior/ov_executable_network/exec_graph_info.hpp

```diff
@@ -496,6 +490,71 @@ TEST_P(OVExecGraphImportExportTest, ieImportExportedFunction) {
     EXPECT_EQ(prc, importedExecNet.GetOutputsInfo()["concat_op1"]->getPrecision());
 }
 
+typedef std::tuple<ov::element::Type,  // Element type
+                   ov::Shape,          // Shape
+                   std::string         // Device name
+                   >
+    OVExecGraphUniqueNodeNamesTestParams;
+
+class OVExecGraphUniqueNodeNamesTest : public testing::WithParamInterface<OVExecGraphUniqueNodeNamesTestParams>,
+                                       public OVCompiledNetworkTestBase {
+public:
+    static std::string getTestCaseName(testing::TestParamInfo<OVExecGraphUniqueNodeNamesTestParams> obj) {
+        ov::element::Type netPrecision;
+        ov::Shape inputShapes;
+        std::string targetDevice;
+        std::tie(netPrecision, inputShapes, targetDevice) = obj.param;
+        std::replace(targetDevice.begin(), targetDevice.end(), ':', '_');
+
+        std::ostringstream result;
+        result << "IS=" << ov::test::utils::vec2str(inputShapes) << "_";
+        result << "netPRC=" << netPrecision.to_string() << "_";
+        result << "targetDevice=" << targetDevice;
+        return result.str();
+    }
+
+    void SetUp() override {
+        ov::Shape inputShape;
+        ov::element::Type netPrecision;
+        std::tie(netPrecision, inputShape, target_device) = this->GetParam();
+        SKIP_IF_CURRENT_TEST_IS_DISABLED();
+
+        APIBaseTest::SetUp();
+
+        ov::ParameterVector params{std::make_shared<ov::op::v0::Parameter>(netPrecision, ov::Shape(inputShape))};
+        auto split_axis_op =
+            std::make_shared<ov::op::v0::Constant>(ov::element::i64, ov::Shape{}, std::vector<int64_t>{1});
+        auto split = std::make_shared<ov::op::v1::Split>(params[0], split_axis_op, 2);
+
+        auto concat = std::make_shared<ov::op::v0::Concat>(split->outputs(), 1);
+
+        ov::ResultVector results{std::make_shared<ov::op::v0::Result>(concat)};
+        fnPtr = std::make_shared<ov::Model>(results, params, "SplitConvConcat");
+    }
+
+protected:
+    std::shared_ptr<ov::Model> fnPtr;
+};
+
+TEST_P(OVExecGraphUniqueNodeNamesTest, CheckUniqueNodeNames) {
```

Comment:
The same question as the previous comment

Reply:
It seems it is duplicated code snippet, i forgot delete it, thank your good findings!

Reply:
done